src/explore-education-statistics-admin/src/pages/release/footnotes/__tests__/ReleaseFootnotesPage.test.tsx

```diff
@@ -0,0 +1,279 @@
+    { id: 'footnote-1', content: 'Footnote 1 content', subjects: {} },
+    { id: 'footnote-2', content: 'Footnote 2 content', subjects: {} },
+    { id: 'footnote-3', content: 'Footnote 3 content', subjects: {} },
+  ];
+
+  describe('with footnotes', () => {
+    beforeEach(() => {
+      permissionService.canUpdateRelease.mockResolvedValue(true);
+      footnoteService.getFootnoteMeta.mockResolvedValue(testFootnoteMeta);
+      footnoteService.getFootnotes.mockResolvedValue(testFootnotes);
+      renderPage();
+    });
+
+    test('renders the page with footnotes', async () => {
+      await waitFor(() => {
+        expect(screen.getByText('Footnotes')).toBeInTheDocument();
+      });
+
+      expect(
+        screen.getByRole('link', { name: 'Create footnote' }),
+      ).toBeInTheDocument();
+      expect(
+        screen.getByRole('button', { name: 'Reorder footnotes' }),
+      ).toBeInTheDocument();
+
+      const footNote1 = within(
+        screen.getByTestId('Footnote - Footnote 1 content'),
+      );
+      expect(footNote1.getByText('Footnote 1 content')).toBeInTheDocument();
+      expect(
+        footNote1.getByRole('link', { name: 'Edit footnote' }),
+      ).toBeInTheDocument();
+      expect(
+        footNote1.getByRole('button', { name: 'Delete footnote' }),
+      ).toBeInTheDocument();
+      expect(
+        footNote1.getByRole('button', {
+          name: 'See matching criteria',
+        }),
+      ).toBeInTheDocument();
+
+      const footNote2 = within(
+        screen.getByTestId('Footnote - Footnote 2 content'),
+      );
+      expect(footNote2.getByText('Footnote 2 content')).toBeInTheDocument();
+      expect(
+        footNote2.getByRole('link', { name: 'Edit footnote' }),
+      ).toBeInTheDocument();
+      expect(
+        footNote2.getByRole('button', { name: 'Delete footnote' }),
+      ).toBeInTheDocument();
+      expect(
+        footNote2.getByRole('button', {
+          name: 'See matching criteria',
+        }),
+      ).toBeInTheDocument();
+
+      const footNote3 = within(
+        screen.getByTestId('Footnote - Footnote 3 content'),
+      );
+      expect(footNote3.getByText('Footnote 3 content')).toBeInTheDocument();
+      expect(
+        footNote3.getByRole('link', { name: 'Edit footnote' }),
+      ).toBeInTheDocument();
+      expect(
+        footNote3.getByRole('button', { name: 'Delete footnote' }),
+      ).toBeInTheDocument();
+      expect(
+        footNote3.getByRole('button', {
+          name: 'See matching criteria',
+        }),
+      ).toBeInTheDocument();
+    });
+
+    test('shows the reordering UI when the reorder button is clicked', async () => {
+      await waitFor(() => {
+        expect(screen.getByText('Footnotes')).toBeInTheDocument();
+      });
+
+      userEvent.click(
+        screen.getByRole('button', { name: 'Reorder footnotes' }),
+      );
+
+      expect(
+        screen.getByRole('button', { name: 'Save order' }),
+      ).toBeInTheDocument();
+
```

Comment:
Combined with the other comment on hiding the 'Create footnote' button, would add assertions before and after clicking the button to prove this no longer renders.